sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -884,6 +914,12 @@ else if (mReleaseDownloader == null || !mReleaseDownloader.isDownloading()) {
                 return;
             }
 
+            /* Do not proceed if automatic check for update is disabled and manual check for update has not been called. */
+            if (mAutomaticCheckForUpdateDisabled && !mManualCheckForUpdateRequested) {
+                AppCenterLog.info(LOG_TAG, "Automatic check for update is disabled. Will not check for update.");
```

Comment:
We probably can improve the wording here (most likely folks have more edits here 😅 )
```suggestion
                AppCenterLog.info(LOG_TAG, "Automatic check for update is disabled. The SDK will not check for update now.");
```